spec/data-model/message.json

```diff
@@ -157,6 +157,7 @@
         "type": { "const": "markup" },
         "kind": { "const": "close" },
         "name": { "type": "string" },
+        "options": { "$ref": "#/$defs/options" },
```

Comment:
Same change here as for the TS; we don't need to have separate markup-open, markup-standalone, and markup-close definitions any more.

Reply:
Done